module/src/components/dropdownItems/dropdownItems.component.tsx

```diff
@@ -17,7 +17,7 @@ export interface IDropdownItem extends IIconWrapperProps<IconSet, IconSet> {
   id: ArmstrongId;
 
   /** props to spread onto the div element for the dropdown item */
-  htmlProps?: Omit<React.DetailedHTMLProps<React.BaseHTMLAttributes<HTMLLIElement>, HTMLLIElement>, 'onMouseUp' | 'ref'>;
+  htmlProps?: Omit<React.DetailedHTMLProps<React.BaseHTMLAttributes<HTMLElement>, HTMLLIElement>, 'onMouseUp' | 'ref'>;
```

Comment:
I may have just missed it, but I can't see where you're actually spreading these props onto the DropdownItem

Reply:
Ahh yeah so it was already there, I see what's going on.

How come you've changed the type to HTMLElement rather than HTMLLiElement?

Reply:
Looking at line 20 in tagInput, you have got a different prop for it, so why would this need to be able to be spread onto a div?

Reply:
The problem is that the AutoCompleteInputMulti uses a tag input and a dropdown, The method 'parseOptionTag' in `autoCompleteInputMulti.component` breaks if I revert that change

Reply:
In the autocompleteinputmulti, I think you'd want two separate props. Like dropdownitemhtmlprops and taghtmlprops

It's unlikely you'd want to add the same html props to both, I think, and keeping them separate solves this

Reply:
good call, this is much more flexible. Done!